lib/project_types/script/messages/messages.rb

```diff
@@ -117,6 +119,10 @@ module Messages
             Usage: {{command:%s enable --API_key=<API_key> --shop_domain=<my_store.myshopify.com>}}
           HELP
 
+          info: "{{*}} A script always remains enabled until you disable it ⁠— even after deploying "\
```

Comment:
🚲 @jacobsteves: I see the styles are encoded in the message file here, and it's done for other project types too.  Just curious, are we sure that words like `green` or `cyan` won't get translated?

Reply:
Good question! I don't think we know that yet. Will have to keep an eye out